sdk/src/main/java/com/horizen/account/api/rpc/types/EthereumReceiptView.java

```diff
@@ -0,0 +1,62 @@
+package com.horizen.account.api.rpc.types;
+
+import com.fasterxml.jackson.annotation.JsonView;
+import com.horizen.account.receipt.EthereumLog;
+import com.horizen.account.receipt.EthereumReceipt;
+import com.horizen.account.transaction.EthereumTransaction;
+import com.horizen.serialization.Views;
+import com.horizen.utils.BytesUtils;
+import org.web3j.utils.Numeric;
+import scala.collection.immutable.List;
+
+@JsonView(Views.Default.class)
+public class EthereumReceiptView {
+    private String transactionHash;
+    private String transactionIndex;
+    private String blockHash;
+    private String blockNumber;
+    private String from;
+    private String to;
+    private String cumulativeGasUsed;
+    private String gasUsed;
+    private String contractAddress;
+    private List<EthereumLog> logs;
+    private String logsBloom;
+    private String root;
+    private String status;
+    private String effectiveGasPrice;
+
+    public EthereumReceiptView() {
+    }
+
+    public EthereumReceiptView(EthereumReceipt receipt, EthereumTransaction tx) {
+        transactionHash = Numeric.toHexString(receipt.transactionHash());
+        transactionIndex = String.valueOf(receipt.transactionIndex());
```

Comment:
do we expect decimal or hexadecimal format here?
I think it should be `Numeric.toHexString...` or `toHexStringWithPrefix`?
same for `blockHash`